src/components/DataTable/DataTable.stories.tsx

```diff
@@ -0,0 +1,26 @@
+import { DataTable, DataTableBody, DataTableEmpty, DataTableHeader } from "."
+
+export default {
+  title: "Components/Data Table",
+}
+
+const Template = {
+  children: "Data Table",
+}
+
+export const Empty = () => <DataTableEmpty />
+Empty.args = { ...Template, children: "Data Table Empty" }
+
+export const dataTable = () => (
+  <DataTable>
+    <DataTableHeader>
+      <td>one</td>
+      <td>two</td>
+    </DataTableHeader>
+    <DataTableBody>
```

Comment:
Shouldn't there be a row as well?

Reply:
Table Body & Table Header have tr's wrapping the children